autograder/tests/data/submitty_config.json

```diff
@@ -0,0 +1,21 @@
+{
+    "submitty": {
+        "submitty_install_dir": "/usr/local/submitty",
+        "submitty_repository": "/usr/local/submitty/GIT_CHECKOUT/Submitty",
+        "submitty_data_dir": "/var/local/submitty",
+        "autograding_log_path": "/var/local/submitty/logs/autograding",
+        "sys_admin_email": "submitty@test.com",
+        "sys_admin_url": "https://test.com",
+        "site_log_path": "/var/local/submitty/logs",
+        "submission_url": "https://submitty.test.com",
+        "vcs_url": "https://gitlab.test.com",
+        "cgi_url": "https://submitty.test.com/cgi-bin",
+        "websocket_port": 8443,
+        "institution_name": "Test University",
+        "username_change_text": "Submitty welcomes individuals of all ages, backgrounds, citizenships, disabilities, sex, education, ethnicities, family statuses, genders, gender identities, geographical locations, languages, military experience, political views, races, religions, sexual orientations, socioeconomic statuses, and work experiences. In an effort to create an inclusive environment, you may specify a preferred name to be used instead of what was provided on the registration roster.",
```

Comment:
This configuration parameter will be removed in the future: https://github.com/Submitty/Submitty/pull/8921

Depending on which PR gets merged first it'll be a small change to remove this so not a big deal and no changes required now but just a heads up